pallets/asset-manager/src/lib.rs

```diff
@@ -292,6 +332,45 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Mint asset by its asset id to a beneficiary.
+		///
+		/// * `origin`: Caller of this extrinsic, the acess control is specfied by `ForceOrigin`.
+		/// * `asset_id`: AssetId to be updated.
+		/// * `beneficiary`: Account to mint the asset.
+		/// * `amount`: Amount of asset being minted.
+		/// # <weight>
+		/// TODO: get actual weight
+		/// # </weight>
+		#[pallet::weight(50_000_000)]
```

Comment:
Please add an associated type `WeightInfo` to `Config`, and then benchmark this pallet, finally update this weight.